src/main/resources/actionsvc/xsd/actionInstruction.xsd

```diff
@@ -65,6 +65,26 @@
 		</xs:sequence>
 	</xs:complexType>
 
+	<!-- a complex type for classifiers-->
+	<xs:complexType name="ActionClassifiers">
+		<xs:sequence>
+			<xs:element name="legalBasis" type="xs:string" minOccurs="0"
+						maxOccurs="1" />
+			<xs:element name="region" type="xs:string" minOccurs="0"
+						maxOccurs="1" />
+			<xs:element name="respondentAccountStatus" type="xs:string" minOccurs="0"
```

Comment:
I think we should separate Classifiers from State. Classifiers are currently just legalBasis and region.